packages/client/utils/taskStatus.ts

```diff
@@ -10,7 +10,7 @@ export const taskStatusLabels: Record<TaskStatus, TaskStatusLabel> = {
   [TaskStatus.PRIVATE]: TaskStatusLabel.PRIVATE
 } as const
 
-export const taskStatusColors: Record<TaskStatus, PALETTE> = {
+export const taskStatusColors: Record<TaskStatus, string> = {
```

Comment:
as PALETTE is not an enum anymore, we'll loose a type safety here 😞

Reply:
good riddance to const enums! long live string unions 🎉